country-map.js

```diff
@@ -139,6 +47,14 @@ module.exports = {
         "abbreviation": "AW",
         "currency": "AWG"
     },
+    "Ascension Island": {
+        "abbreviation": "AC",
+        "currency": "USD"

```

Comment:
per https://en.wikipedia.org/wiki/Pound_sterling_in_the_South_Atlantic_and_the_Antarctic this should probably be GBP.